presto-hive/src/main/java/io/prestosql/plugin/hive/HiveMetadata.java

```diff
@@ -1748,6 +1751,32 @@ public void dropView(ConnectorSession session, SchemaTableName viewName)
         return tableNames.build();
     }
 
+    @Override
+    public Map<String, Object> getSchemaProperties(ConnectorSession session, CatalogSchemaName schemaName)
+    {
+        checkState(filterSchema(schemaName.getSchemaName()), "Schema is not accessible: %s", schemaName);
```

Comment:
We have discussed this above :) See @findepi 's comment. It's not access control.

This is only for the `information_schema` and `sys`. And those are not available for properties or ownership. These should never be triggered, since all callers check for schemaExists first, it's just a safety consistency check in case someone adds a call to this later.

Reply:
I see, thanks!